themes/kiali/static/css/kiali-home.css

```diff
@@ -40,22 +40,24 @@ section#hero p.buttons {
   margin-top: var(--kiali-blue-spacing);
 }
 
-@media only screen and (min-width: 850px) {
-  section#hero {
-    padding-top: var(--kiali-bright-green-spacing);
-    padding-bottom: var(--kiali-bright-green-spacing);
-  }
-}
-
 @media only screen and (min-width: 430px) {
   section#hero p.buttons a.button {
     display: inline-block;
     margin-bottom: 0;
   }
 }
 
-
 /* "About" block styles */
+#about {
+  overflow: hidden;
+}
+
+#about-description {
+  max-width: 600px;
+}
+#intro-video {
+  float: right;
```

Comment:
The video cannot float at all resolutions. It needs to fallback to be placed at the bottom of the about text on small width.
It doesn't look good if you visit the website on a phone in landscape orientation. It also looks bad if you have two browser windows side by side -- you may want to see Kiali and Istio websites at the same time ;)

Reply:
You also may need to add margins to the left of the video. On not too wide screens the text is too close to the video:
![image](https://user-images.githubusercontent.com/23639005/68346912-fdc9cf80-00ba-11ea-980c-16da0c5f6e97.png)